packages/core/strapi/src/cli/commands/transfer/command.ts

```diff
@@ -44,14 +44,122 @@ const command = () => {
       .hook('preAction', validateExcludeOnly)
       .hook(
         'preAction',
-        ifOptions(
-          (opts) => !(opts.from || opts.to) || (opts.from && opts.to),
-          async () =>
+        async (thisCommand) => {
+          const opts = thisCommand.opts();
+          
+          // If neither --from nor --to are provided, prompt for transfer direction
+          if (!(opts.from || opts.to)) {
+            // Log environment variables status
+            const hasUrl = Boolean(process.env.STRAPI_TRANSFER_URL);
+            const hasToken = Boolean(process.env.STRAPI_TRANSFER_TOKEN);
+
+            console.info('ℹ️  Data transfer documentation: https://docs.strapi.io/dev-docs/data-management/transfer');
+
+            if (hasUrl || hasToken) {
+              console.info('ℹ️  Found transfer configuration in your environment:');
+              if (hasUrl) {
+                console.info('   → STRAPI_TRANSFER_URL environment variable value will be used as default URL');
+              }
+              if(hasToken) {
+                console.info('   → STRAPI_TRANSFER_TOKEN environment variable value will be used as default token');
+              }
+            } else {
+              console.info('ℹ️  No transfer configuration found in environment variables');
+              console.info('   → Add STRAPI_TRANSFER_URL and STRAPI_TRANSFER_TOKEN environment variables to make the transfer process faster for future runs');
+            }
+            console.log(); // Empty line for better readability
```

Comment:
```suggestion
            console.info(); // Empty line for better readability
```
Since we used .info above